psyneulink/core/llvm/execution.py

```diff
@@ -503,13 +503,13 @@ def _initialize_autodiff_param_struct(self, autodiff_stimuli):
         targets = autodiff_stimuli.get("targets", {})
         epochs = autodiff_stimuli.get("epochs", 0)
 
-        num_inputs = len(next(iter(inputs.values())))
+        num_trials = len(next(iter(inputs.values())))
```

Comment:
This is a wrong rename please revert it back to `nump_inputs`.
In general, the number of inputs is different from the number of trials (even if it's the same for autodiff),
you can have 10 trials with one input or 8 trials with 4 inputs.